src/applications/edu-benefits/10203/containers/IntroductionPage.jsx

```diff
@@ -2,29 +2,50 @@ import React from 'react';
 import { focusElement } from 'platform/utilities/ui';
 import OMBInfo from '@department-of-veterans-affairs/formation-react/OMBInfo';
 import FormTitle from 'platform/forms-system/src/js/components/FormTitle';
-import SaveInProgressIntro from 'platform/forms/save-in-progress/SaveInProgressIntro';
+import SaveInProgressIntro from '../content/SaveInProgressIntro';
+import { connect } from 'react-redux';
 
 export class IntroductionPage extends React.Component {
   componentDidMount() {
     focusElement('.va-nav-breadcrumbs-list');
   }
+
+  loggedIn() {
+    return this.props.isLoggedIn ? (
+      <SaveInProgressIntro
+        prefillEnabled={this.props.route.formConfig.prefillEnabled}
+        messages={this.props.route.formConfig.savedFormMessages}
+        pageList={this.props.route.pageList}
+        startText="Start the education application"
+      />
+    ) : (
+      <SaveInProgressIntro
+        buttonOnly
+        prefillEnabled={this.props.route.formConfig.prefillEnabled}
+        messages={this.props.route.formConfig.savedFormMessages}
+        pageList={this.props.route.pageList}
+        startText="Start the education application"
+      />
+    );
+  }
+
   render() {
     return (
       <div
         className="schemaform-intro"
         itemScope
         itemType="http://schema.org/HowTo"
       >
-        <FormTitle title="Manage your education benefits" />
+        <FormTitle title="Apply for the Rogers STEM Scholarship" />
         <p itemProp="description">
-          Equal to VA Form 22-10203 (Request for Change of Program or Place of
-          Training).
+          Equal to VA Form 22-10203 (Application for Edith Nourse Rogers STEM
+          Scholarship).
         </p>
         <SaveInProgressIntro
```

Comment:
should this be replaced by `{this.loggedIn()}` so that prompt is consistent throughout page?

Reply:
The top component is consistent, but the bottom conditionally render based off if the user is logged in or not.